docs/start/index.mdx

```diff
@@ -1,6 +1,6 @@
 ---
 title: Introduction
-description: Introduction to IBM Quantum documentation
+description: Introduction to IBM Quantum, Qiskit, and Qiskit Runtime.
```

Comment:
I think this one should still mention the word documentation right? The page itself mostly explains what the docs platform is more than anything else

Reply:
Done.